### ISSUE TITLE
Fix /tmp references for Windows

### ISSUE THREAD

**Issue by bslatkin** [bslatkin/effectivepython#36]
Item 3, page 7 and Item 13, page 26. Windows complains about /tmp. It's easier just to use the current directory, like this:

```
handle = open('random_data.txt')  # May raise IOError
try:
    data = handle.read()  # May raise UnicodeDecodeError
finally:
    handle.close()        # Always runs after try:
```

**Comment by bslatkin** [comment]
Addressed a bunch of these in 4a0a3b643077eea0bd2843c981f437c40a998560

**Comment by bslatkin** [comment]
Fixed for the next printing.
